saleor/order/actions.py

```diff
@@ -163,6 +163,16 @@ def order_voided(order: "Order", user: "User", payment: "Payment"):
     get_plugins_manager().order_updated(order)
 
 
+def order_returned(
+    order: "Order",
+    user: Optional["User"],
+    returned_lines: List[Tuple[QuantityType, OrderLine]],
+):
+    order_returned_event(order=order, user=user, returned_lines=returned_lines)
+    update_order_status(order)
+    get_plugins_manager().order_updated(order)
```

Comment:
`manager` could be passed as a param to this function; it's already fetched in the `ordered_fulfilled` function.